doc/chomp_interface/scripts/collision_scene_example_cluttered.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+import rospy
+from moveit_commander import RobotCommander, PlanningSceneInterface
+import geometry_msgs.msg
+import time
+
+
+class CollisionSceneExampleCluttered(object):
+
+    def __init__(self):
+        self._scene = PlanningSceneInterface()
+
+        # clear the scene
+        self._scene.remove_world_object()
+
+        self.robot = RobotCommander()
+
+        # pause to wait for rviz to load        
+        print "============ Waiting while RVIZ displays the scene with four obstacles..."
+        rospy.sleep(4)
```

Comment:
Hi Mike, if I don't put any sleep, the obstacles do not show up in RViz, hence I used a sleep over here, I could reduce it to rospy.sleep(2), any reasons for not using sleep ?

Reply:
Sleep in this context is a hack that masks the bigger issue. Instead of blindly sleeping, this demo script should wait for the correct topics to be available before publishing to the planning scene interface and it should wait for those changes to be reflected in the planning scene before returning.

Reply:
Waiting an arbitrary number of seconds is not an approach that we want to encourage among the users of MoveIt!